arkouda/series.py

```diff
@@ -75,28 +77,55 @@ class Series:
 
     Parameters
     ----------
-    ar_tuple : 2-tuple of arkouda arrays with the first being the grouping key(s) and the
-             second being the value. The grouping key(s) will be the index of the series.
-
+    index : pdarray, Strings
+        an array of indices associated with the data array.
+        If empty, it will default to a range of ints whose size match the size of the data.
+        optional
+    data : Tuple, List, groupable_element_type
+        a 1D array. Must not be None if ar_tuple is not provided.
+
+    Raises
+    ------
+    TypeError
+        Raised if index is not a pdarray or Strings object
+        Raised if data is not a pdarray, Strings, or Categorical object
+    ValueError
+        Raised if the index size does not match data size
+
+    Notes
+    -----
+    The Series class accepts either positional arguments or keyword arguments.
+    If entering positional arguments,
+        2 arguments entered:
+            argument 1 - index
+            argument 2 - data
```

Comment:
I think these got swapped at some point. It looks to me like `data` should be first now

Reply:
resolving these, they will be covered in the reformatting